packages/metastream-app/src/locale/ar.ts

```diff
@@ -28,72 +28,72 @@ export default {
   disconnect: 'قطع الاتصال',
   disconnected: 'انقطع الاتصال',
   dj: 'DJ',
-  donate: 'تبرع',
+  donate: 'patreon الدعم على',
```

Comment:
I'd prefer for this string to be generic, without "patreon". That way if I ever decide to switch platforms, this wouldn't need to be changed.

Reply:
alright and sry i was offline